testthat.R

```diff
@@ -16,5 +16,5 @@ check_gpu <- function() {
 if (length(args)==1) {
   testthat::test_file(paste("/input/tests", args[1], sep="/"))
 } else {
-  testthat::test_dir("/input/tests")
+  testthat::test_dir("/input/tests", stop_on_failure=TRUE)
```

Comment:
Isn't it better to see a list of all the failing tests instead of only the first test to fail?